lib/rubocop/directive_comment.rb

```diff
@@ -6,7 +6,9 @@ module RuboCop
   # cops it contains.
   class DirectiveComment
     # @api private
-    REDUNDANT_COP = 'Lint/RedundantCopDisableDirective'
+    REDUNDANT_COP_DEPARTMENT = 'Lint'
```

Comment:
I really think we should name this `REDUNDANT_DIRECTIVE_COP...`, otherwise the name is both funny and confusing. :-) I know that's not related to the work you've done, but I was just amused by reading the diff - the name implies it's some redandant cop. :D